alpha/apps/kaltura/lib/kUploadTokenMgr.php

```diff
@@ -201,10 +211,24 @@ public function uploadFileToToken($fileData, $resume = false, $resumeAt = -1)
 		
 		$this->_uploadToken->setUploadedFileSize($fileSize);
 		$this->_uploadToken->setDc(kDataCenterMgr::getCurrentDcId());
+		$this->_uploadToken->save();
+	}
+	
+	public function uploadedLockImpl($fileSize)
+	{
+		$dbUploadToken = UploadTokenPeer::retrieveByPK($this->_uploadToken->getId());
+		if ($dbUploadToken->getStatus() == UploadToken::UPLOAD_TOKEN_FULL_UPLOAD)
```

Comment:
use allowed status like in
alpha/apps/kaltura/lib/kUploadTokenMgr.php:126
also worth moving to function to avoid duplication